internal/scheduler/preempting_queue_scheduler_test.go

```diff
@@ -1938,6 +1938,16 @@ func TestPreemptingQueueScheduler(t *testing.T) {
 					assert.True(t, ok)
 					assert.NotEmpty(t, nodeId)
 
+					node, err := nodeDb.GetNode(nodeId)
+					require.NoError(t, err)
+					assert.NotEmpty(t, node)
+
+					// Check that the job can actually go onto this node.
+					matches, reason, err := nodedb.StaticJobRequirementsMet(node.Taints, node.Labels, node.TotalResources, jctx)
```

Comment:
Adding this caused an existing test to fail. Copying over the tolerations when evicting made that test pass.